### ISSUE TITLE
Update Danish translations

### ISSUE THREAD

**Issue by Yacobolo** [squidfunk/mkdocs-material#5663]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "da",
  "action.edit": "Redigér denne side",
  "action.skip": "Gå til indholdet",
  "action.view": "Vis kildetekst på denne side",
  "announce.dismiss": "Vis ikke dette igen",
  "blog.archive": "Arkiv",
  "blog.categories": "Kategorier",
  "blog.categories.in": "i",
  "blog.continue": "Læs mere",
  "blog.draft": "Udkast",
  "blog.index": "Gå tilbage",
  "blog.meta": "Metadata",
  "blog.references": "Relateret indhold",
  "clipboard.copy": "Kopiér til udklipsholderen",
  "clipboard.copied": "Kopieret til udklipsholderen",
  "consent.accept": "Accepter",
  "consent.manage": "Administrer indstillinger",
  "consent.reject": "Afvis",
  "footer": "Sidefod",
  "footer.next": "Næste",
  "footer.previous": "Forrige",
  "header": "Sidehoved",
  "meta.comments": "Kommentarer",
  "meta.source": "Kilde",
  "nav": "Navigation",
  "readtime.one": "1 minuts læsetid",
  "readtime.other": "# minuts læstid",
  "rss.created": "RSS feed",
  "rss.updated": "RSS feed af opdateret indhold",
  "search": "Søg",
  "search.placeholder": "Søg",
  "search.share": "Del",
  "search.reset": "Nulstil søgning",
  "search.result.initializer": "Start søgning",
  "search.result.placeholder": "Indtast søgeord",
  "search.result.none": "Ingen resultater fundet",
  "search.result.one": "1 resultat",
  "search.result.other": "# resultater",
  "search.result.more.one": "1 resultat mere på denne side",
  "search.result.more.other": "# resultater mere på denne side",
  "search.result.term.missing": "Mangler",
  "select.language": "Vælg sprog",
  "select.version": "Vælg version",
  "source": "Åbn arkiv",
  "source.file.contributors": "Bidragydere",
  "source.file.date.created": "Oprettet",
  "source.file.date.updated": "Sidste ændring",
  "tabs": "Faner",
  "toc": "Indholdsfortegnelse",
  "top": "Tilbage til start"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Thanks! Added in 5dc599803.

**Comment by squidfunk** [comment]
Released as part of [9.1.18](https://squidfunk.github.io/mkdocs-material/changelog/#9.1.18)

**Comment by cfaagaard** [comment]
Nice job!! One small thing. This line:

 "readtime.other": "# minuts læstid",

should be:

 "readtime.other": "# minuts læsetid"

This PR has the change https://github.com/squidfunk/mkdocs-material/pull/6133
